packages/x-components/src/x-modules/related-prompts/wiring.ts

```diff
@@ -34,7 +34,12 @@ const setRelatedPromptsQueryFromPreview = wireCommit(
 const setRelatedPromptQuery = wireCommit('setSelectedQuery');
 /** Sets the related prompts state `selectedPrompt`. */
 const setRelatedPrompt = wireCommit('setSelectedPrompt');
-
+/**
+ * Sets the related prompts state `relatedTags`.
+ *
+ * @public
+ */
+export const setRelatedPromptsRelatedTags = wireCommit('setRelatedPromptsRelatedTags');
```

Comment:
```suggestion
/** Sets the related prompts state `relatedTags`. */
 const setRelatedPromptsRelatedTags = wireCommit('setRelatedPromptsRelatedTags');
```

It's not needed to expose it, right?

Reply:
I have already tried to remove the `@public`, but when I run the build, it fails because the `@public` label is missing...